CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Summoner
 
+1.0.7
+=====
+
+* Add stackage badges
+* Fix travis cache
```

Comment:
Could you please write changelog according to format? See examples below. Basically, add link to the issue and write short description on the next line.